### DISCUSSION TITLE
How Broken are we Talking When Using Optifine?

### DISCUSSION THREAD

**Discussion by SwissxCore** [terrarium-earth/Ad-Astra#298] [Q&A] [answered]
So I just noticed the "Breaks with Optifine" graphic on the description page on CurseForge, and I wanted to know exactly "how" broken we're talking. Is it literally unplayable? Causing crashes? Or is it just a few visual bugs like flickering or something?

I'm just struggling to move away from Optifine as there's a resource pack I'm using that makes use of it by allowing models to be changed and random textures to be selected. Maybe I'm asking the wrong question; would Rubidium afford the same resource pack support?

Thank you.

**Comment by CodexAdrian** [comment, marked as answer]
you will not be able to use ad astra with optifine, it breaks the mod alot. all of the energy and fluid in machines no longer function properly, so its literally unplayable. we have no plans to support optifine, honestly optifine breaks half the mods out there, id get used to the idea that optifine just does not fit in the world of modern minecraft modding and find replacements as quick as possible

**Reply by SwissxCore** [reply]
Yeah, I somewhat figured that was the case. I'll never ask someone to support something they don't want to support, especially with how difficult it is to fix issues regarding Optifine. Optifine has just been around for so long that it's become heavily ingrained into Minecraft modding, and it isn't going to vanish over night.

Either way, I really appreciate the quick response, I'll take a closer look at Rubidium to see what it's capable of; I don't think it's going to be a 1 to 1 replacement for Optifine, but it can hopefully offer some good framerate improvements.
